proposals/theme_repo.md

```diff
@@ -0,0 +1,102 @@
+# Theme repo standard proposal
+
+## Description Of Issue
+
+Theme repositories are not standardised, outside of any de facto standards from theme loaders
```

Comment:
Which theme loaders have de-facto standards?

Reply:
Cumstain pretty much, it uses this format which isn't yet standardised.

Reply:
Okay, noted.